cunumeric/module.py

```diff
@@ -675,6 +712,8 @@ def linspace(
     delta = stop - start
     y = arange(0, num, dtype=dt)
 
+    out: tuple[Any, ...]  # EllipsisType not even in typing_extensions yet
```

Comment:
Also this comment is gone :shrug: 

https://github.com/nv-legate/cunumeric/pull/413/commits/67f87a7d66a2418f8af6c051a32803ff35e0faca